pages/linux/debugfs.md

```diff
@@ -0,0 +1,27 @@
+# debugfs
+
+> An interactive ext2/ext3/ext4 filesystem debugger.
+
+- Open the filesystem in read only mode:
+
+`debugfs {{/dev/sdXN}}`
+
+- Open the filesystem in read write mode:
+
+`debugfs -w {{/dev/sdXN}}`
+
+- Read commands from cmd_file file, execute them and exit:
+
+`debugfs -f cmd_file`
```

Comment:
Missing token syntax? Also, perhaps this would be more consistent with other existing pages?
```suggestion
`debugfs -f {{path/to/cmd_file}}`
```